proto/rpc/v1/tx.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package rpc.v1;
+
+import "rpc/v1/transaction.proto";
+import "rpc/v1/meta.proto";
+
+message TxRequest {
+    // the hash of the transaction
+    bytes hash = 1;
+
+    // if true, return data in binary format
+    // TODO: does this make sense for protobuf?
+    bool binary = 2;
+}
+
+message TxResponse {
+
+    // The actual transaction
+    oneof sttx {
+        Transaction tx = 1;
+        bytes tx_bytes = 2;
+    };
+    // Sequence number of ledger that contains this transaction
+    uint64 ledger_index = 3;
+
+    // Hash of the transaction
+    bytes hash = 4;
+
+    // whether the ledger has been validated
+    bool validated = 5;
+
+    // metadata about the transaction
+    oneof stmeta {
```

Comment:
ditto

Reply:
What do you think is a good name for this? I could call it serialized_meta.

Reply:
changed to serialized_meta